src/java.base/share/classes/java/util/Objects.java

```diff
@@ -164,6 +164,24 @@ public static String toString(Object o, String nullDefault) {
         return (o != null) ? o.toString() : nullDefault;
     }
 
+    /**
+     * {@return a string equivalent to the string returned by {@code
+     * Object.toString} if that method is not overridden}
+     *
```

Comment:
The original Object.toString() invokes the hashCode() method.
Perhaps the spec should indicate that it calls IdentityhashCode() instead of hashCode().

Though not incorrect, in Valhalla, Not all objects have identity and will need to different implementation.

Are there any use cases in the JDK that will use this method?

Reply:
I updated the return statement to say if hashCode isn't overridden either. The implSpec tag does explicitly mention identityHashCode already.

There are a few instances of this sort of code in the JDK src directory already; I may include a refactoring of at least one of them in a future revision of the PR. Thanks.